tsec-libsodium/src/test/scala/tsec/libsodium/GenericHash.scala

```diff
@@ -0,0 +1,71 @@
+package tsec.libsodium
+
+import cats.effect.IO
+import tsec.common._
+import tsec.libsodium.hashing._
+import fs2._
+import tsec.libsodium.hashing.internal.SodiumHashPlatform
+
+class GenericHash extends SodiumSpec {
+
+  def hashTest[A](platform: SodiumHashPlatform[A]) = {
+    behavior of "Sodium hash for " + platform.algorithm
+
+    it should "hash two bytes to equal hashes" in {
```

Comment:
vaguely misleading description.

Reply:
pretty sure my brain was fried when I wrote that.

I will fixu